### DISCUSSION TITLE
How to manually interact with Prisma?

### DISCUSSION THREAD

**Discussion by sara-dubois** [prisma/prisma#9436] [Q&A] [answered]
Is there any official Prisma way to manually interact with Prisma? I'm thinking something like:
- [blitz console](https://blitzjs.com/docs/cli-console)
- [prisma-repl](https://github.com/egoist/prisma-repl)

This is perhaps a stupid question, but if there's no REPL, how do I try out the Prisma API, or do simple manual one-off operations? (Couldn't get prisma-repl to work)

**Comment by ryands17** [comment, marked as answer]
@sara-dubois 👋 

Unfortunately there isn't any official way to manually interact with Prisma as of now. We mostly suggest the above tools for a REPL like environment. What was the issue that occurred with `prisma-repl`?

**Comment by 2color** [comment]
Hey Sarah,

Check out the [Prisma Data Platform](cloud.prisma.io) where you have the query console that allows you to test out Prisma Client queries directly from the browser.

[Check out this demo](https://youtu.be/nLHJCzWxS5M?t=1719) from the last What's new in Prisma Livestream.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
